src/display/editor/tools.js

```diff
@@ -687,7 +689,8 @@ class AnnotationEditorUIManager {
     );
   }
 
-  constructor(container, eventBus, pdfDocument, pageColors) {
+  constructor(viewer, container, eventBus, pdfDocument, pageColors) {
+    this.#viewer = viewer;
     this.#container = container;
```

Comment:
Nit: Please keep the same ordering as the viewer does.
```suggestion
  constructor(container, viewer, eventBus, pdfDocument, pageColors) {
    this.#container = container;
    this.#viewer = viewer;
```